docs/en_US/network-dumps.md

```diff
@@ -18,14 +18,13 @@ In order to make this easier, we have developed a suite of internal tools to cap
 </div>
 
 # Submissions
-There are multiple ways to submit network dumps. You may either contact a developer directly, or upload your network dumps to any channel in the Discord server. If uploaded to Discord, the Bandwidth bot will automatically detect the dumps and reupload them to our private channel, deleting the original message for security purposes.
+There are two ways to submit network dumps. You may either contact a developer directly, or use the `/upload-network-dump` command on our Discord server.
 
 When submitting we ask that you do the following:
 
-1. Only submit dumps for one game at a time, to keep the files organized. Sending multiple dumps of multiple games makes it difficult to know which dumps go to which game,
-2. Perform only small actions if possible. Each session should be focussed on as few aspects of the game as possible to minimize "noise" (unrelated packets) which makes it difficult to decipher what is actually going on. For example, rather than uploading a session of playing random courses on Super Mario Maker it would be more helpful if the session was focussed on one part of the game, such as 100 Mario Challenge, starring a course, etc. That way it's clear exactly what the packets are doing in game.
-3. When submitting, please provide a detailed description of what was done in the session. For example "Loaded course world and played a random course, starred the course and played a recommended course". This will add further context as to what the packets are doing in the dumps.
-4. When submitting, please rename the dumps to something meaningful. Such as `played-and-starred-course-smm-jpn.pcap` rather than `did-something.pcap` or `12-20-23.pcap`. This does not apply to HokakuCafe BIN files, as they must remain named as they are for the Discord bot to detect them
+1. Perform as few actions as possible per session. Each session should be focussed on as few aspects of the game as possible to minimize "noise" (unrelated packets) which makes it difficult to decipher what is actually going on. For example, rather than uploading a session of playing random courses on Super Mario Maker it would be more helpful if the session was focussed on one part of the game, such as 100 Mario Challenge, starring a course, etc. That way it's clear exactly what the packets are doing in game.
+2. When submitting, please provide a detailed description of what was done in the session. For example "Loaded course world and played a random course, starred the course and played a recommended course". This will add further context as to what the packets are doing in the dumps.
+3. When submitting, please rename the dumps to something meaningful. Such as `played-and-starred-course-smm-jpn.pcap` rather than `did-something.pcap` or `12-20-23.pcap`. This does not apply to HokakuCafe BIN files, as they must remain named as they are. The file name of HokakuCafe BIN files gives us your NEX username (PID).
```

Comment:
> The file name of HokakuCafe BIN files gives us your NEX username (PID).

Well tecnically we can get the PID from the `Login`/`LoginEx` requests. Having the game server ID is useful though

Reply:
True. Do you suggest we change this?

Reply:
Maybe something like "The file name of HokakuCafe BIN files gives us more information about the game servers."?

Reply:
Updated :+1: